src/analysis/retail/hunter/survival/modules/apl/AplCheck.tsx

```diff
@@ -0,0 +1,196 @@
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/hunter';
+import RESOURCE_TYPES from 'game/RESOURCE_TYPES';
+import SpellLink from 'interface/SpellLink';
+import { suggestion } from 'parser/core/Analyzer';
+import { AnyEvent } from 'parser/core/Events';
+import aplCheck, { build, CheckResult, PlayerInfo, tenseAlt } from 'parser/shared/metrics/apl';
+import annotateTimeline from 'parser/shared/metrics/apl/annotate';
+import { AplRuleProps } from 'parser/shared/metrics/apl/ChecklistRule';
+import * as cnd from 'parser/shared/metrics/apl/conditions';
+
+const withTip = cnd.buffPresent(SPELLS.TIP_OF_THE_SPEAR_CAST);
+const hasFocus = (min: number) => cnd.hasResource(RESOURCE_TYPES.FOCUS, { atLeast: min });
+export const apl = build([
+  {
+    spell: TALENTS.KILL_COMMAND_SURVIVAL_TALENT,
+    condition: cnd.describe(
+      cnd.and(
+        cnd.hasTalent(TALENTS.RELENTLESS_PRIMAL_FEROCITY_TALENT),
+        cnd.buffMissing(SPELLS.TIP_OF_THE_SPEAR_CAST),
+        cnd.buffPresent(SPELLS.COORDINATED_ASSAULT_BUFF),
+      ),
+      (tense) => <>you have 0 tip stacks & Relentless Primal Ferocity is active</>,
+    ),
+  },
+  {
+    spell: TALENTS.RAPTOR_STRIKE_TALENT,
+    condition: cnd.describe(
+      cnd.or(
+        //cnd.debuffMissing(SPELLS.SERPENT_STING_SURVIVAL,),
+        cnd.buffRemaining(SPELLS.HOWL_OF_THE_PACK_BUFF, 8, { atMost: 1400 }, false),
+      ),
+      (tense) => {
+        //const isSerpentStingMissing = cnd.debuffMissing(SPELLS.SERPENT_STING_SURVIVAL);
+        const isHowlOfThePackFading = cnd.not(
+          cnd.buffRemaining(SPELLS.HOWL_OF_THE_PACK_BUFF, 8, { atLeast: 1400 }, false),
+        );
+
+        // if (isSerpentStingMissing) {
+        //   return <>Serpent Sting is missing</>;
+        // } else
+        if (isHowlOfThePackFading) {
```

Comment:
Was struggling to get it to show anything other than the Serpent Sting missing so that explains why.
 I was trying to avoid having a lot of separate APL lines and get a single priority to display what  they did incorrectly at the time since these 2 basically will never conflict in actual play and are your highest priority action if either is true outside of RPF CA. Would it be better to just go with multiple lines and just accept that there will be some bloat to the list? I had this same issue with wildfire bomb below where the APL line is a single line and so they're all of equal importance but end up being split into 3 different lines in the APLchecker.